sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala

```diff
@@ -591,6 +592,8 @@ object DataSource extends Logging {
       "org.apache.spark.sql.execution.datasources.orc" -> nativeOrc,
       "org.apache.spark.ml.source.libsvm.DefaultSource" -> libsvm,
       "org.apache.spark.ml.source.libsvm" -> libsvm,
+      "org.apache.spark.ml.source.image.ImageFileFormat.DefaultSource" -> image,
```

Comment:
Since this is new datasource, I think we wouldn't probably need this in backword compatibility map.